gears/src/rest/pagination.rs

```diff
@@ -9,12 +9,20 @@ pub struct Pagination {
     /// offset is a numeric offset that can be used when key is unavailable.
     /// It is less efficient than using key. Only one of offset or key should
     /// be set.
+    #[serde(alias = "pagination.offset")]
     offset: Option<u32>,
     /// limit is the total number of results to be returned in the result page.
     /// If left empty it will default to a value to be set by each app.
+    #[serde(alias = "pagination.limit")]
```

Comment:
I'm unsure if `alias` attribute overrides default name. If so, would it be better to add the original name too?

Reply:
It doesn't override the original name. It allows to deserialize a struct with such name into destination struct. I see no reasons for additional alias because it is a default value.